compiledtrees/code_gen.py

```diff
@@ -138,11 +139,18 @@ def code_gen_ensemble(trees, individual_learner_weight, initial_value,
     if gen is None:
         gen = CodeGenerator()
 
+    tree_files = []
```

Comment:
Could you make this a function and a list comprehension to avoid the mutable list/append?